tsparser/src/parser/resources/apis/gateway.rs

```diff
@@ -39,7 +39,7 @@ pub const GATEWAY_PARSER: ResourceParser = ResourceParser {
         for r in iter_references::<Res>(&module, &names) {
             let r = report_and_continue!(r);
 
-            let object = match &r.bind_name {
+            let object = match r.bind_name.ident() {
                 None => None,
```

Comment:
I guess this could resolve the default export, but not sure how difficult that would be to do. If it's tricky maybe add a TODO here?

Reply:
I've moved all these to a separate function and are now resolving it via `module.data`. The object will still resolve to `None` for default export expressions, I've added a todo for that for now, maybe that is fine to start with?